posts/2025-11-13-numpy-tutorials-jb2.md

```diff
@@ -0,0 +1,22 @@
+---
+title: NumPy tutorials now built with Jupyter Book 2
+date: 2025-11-13
+authors:
+  - name: Jupyter Book Team
+---
+
+The [NumPy tutorials site](https://numpy.org/numpy-tutorials/) has upgraded to Jupyter Book 2! The [migration was just merged](https://github.com/numpy/numpy-tutorials/pull/274), bringing MyST-powered documentation to one of scientific Python's foundational libraries.
+
+This work came out of the [Scientific Python Developer Summit in Copenhagen](https://scientific-python.org/summits/developer/2025-nov/) in November 2025. The summit created space for NumPy contributors to share experiences with the broader ecosystem and tackle the migration as a focused, right-sized project.
+
+Many projects are hesitant to move from Sphinx. NumPy tutorials are a great fit for JB2 because they focus on narrative content and [don't need API documentation](https://github.com/jupyter-book/mystmd/issues/1259). Seeing NumPy make this transition is a good sign that JB2 is ready for production use in major open source projects.
```

Comment:
(somehow I cannot suggest changes? I'm not sure what's going on with my github)

Maybe rephrase: 

"While many projects are hesitant to move on from JB1, the Numpy tutorials maintainers are agreed to make the jump for JB2 because the material focuses on narrative and notebook content and [don't need API documentation](https://github.com/jupyter-book/mystmd/issues/1259).